artifacts/openapi.yaml

```diff
@@ -612,17 +612,27 @@ components:
           type: string
           pattern: ^[\sa-zA-Z0-9-_()><\[\]]+$
           x-unique: global
+        dhcp_v4interface:
```

Comment:
Does dhcpv4_interfaces conform to other such names e.g. ipv4_interfaces better than dhcp_v4interfaces? One convention should be good